packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.tsx

```diff
@@ -103,12 +105,24 @@ const PatientSearchLaunch: React.FC<PatientSearchLaunchProps> = () => {
     setOpen(false);
   }, []);
 
+  const hidePanel = useCallback(() => {
```

Comment:
And then you'll probably want to rename the `handleCloseSearchPanel` function to like `handleCloseSearchInput`. And then change the name of `hidePanel` to `handleCloseResultsPanel`. Something like that.